### ISSUE TITLE
 javax.management.MBeanServer

### ISSUE THREAD

**Issue by VedNirmal** [spring-projects/spring-boot#3336]
hey,

I am deploying the Spring Boot application on WebLogic 12.1.3 Server. 

On Startup I'm getting the below exception : 
Caused By: java.lang.IllegalArgumentException: Object of class [null] must be an instance of interface javax.management.MBeanServer

even I'm not using any code relating to MBeanServer.

Could anyone help on this.

Thanks,
Nirmal

aused By: java.lang.IllegalArgumentException: Object of class [null] must be an instance of interface javax.management.MBeanServer
	at org.springframework.util.Assert.isInstanceOf(Assert.java:339)
	at org.springframework.util.Assert.isInstanceOf(Assert.java:319)
	at org.springframework.context.annotation.MBeanExportConfiguration$SpecificPlatform.getMBeanServer(MBeanExportConfiguration.java:150)
	at org.springframework.context.annotation.MBeanExportConfiguration.setupServer(MBeanExportConfiguration.java:111)
	at org.springframework.context.annotation.MBeanExportConfiguration.mbeanExporter(MBeanExportConfiguration.java:85)
	at org.springframework.context.annotation.MBeanExportConfiguration$$EnhancerBySpringCGLIB$$66bc3073.CGLIB$mbeanExporter$3(<generated>)
	at org.springframework.context.annotation.MBeanExportConfiguration$$EnhancerBySpringCGLIB$$66bc3073$$FastClassBySpringCGLIB$$540bc275.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:312)
	at org.springframework.context.annotation.MBeanExportConfiguration$$EnhancerBySpringCGLIB$$66bc3073.mbeanExporter(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

**Comment by jhoeller** [comment]
You seem to be using an old version of Spring Framework there, since the corresponding code in your stacktrace has been removed as of https://jira.spring.io/browse/SPR-11877... Could you try upgrading to a more recent version, i.e. 4.1+ or 4.0.9?

**Comment by snicoll** [comment]
We prefer to keep this issue track for Spring Boot bugs and enhancements. This sort of question is better-suited to Stackoverflow. In any case, please make sure to [read the release notes](https://github.com/spring-projects/spring-boot/wiki) and adapt your Spring Framework version if necessary.

**Comment by VedNirmal** [comment]
I really appreciate your help.
Upgrading to Spring Version 4.1.X worked fine.

Thanks much!

-Nirmal

**Comment by VedNirmal** [comment]
Hello, 

It was working fine but after a day when I started it start giving below error : 

I tried deleting all temp files still not worked.

Any suggestion? 

Error 404--Not Found

From RFC 2068 Hypertext Transfer Protocol -- HTTP/1.1:

10.4.5 404 Not Found

The server has not found anything matching the Request-URI. No indication is given of whether the condition is temporary or permanent.

If the server does not wish to make this information available to the client, the status code 403 (Forbidden) can be used instead. The 410 (Gone) status code SHOULD be used if the server knows, through some internally configurable mechanism, that an old resource is permanently unavailable and has no forwarding address.
